tasks/main.yml

```diff
@@ -26,7 +28,9 @@
 
 - name: Build base URL
   set_fact:
-    caddy_base_url: "https://{{ caddy_license_account_id }}:{{ caddy_license_api_key }}@caddyserver.com/download/linux/{{ go_arch }}"
+    caddy_base_url: "https://{{ caddy_auth_info }}@caddyserver.com/download/linux/{{ go_arch }}"
+  vars:
+    caddy_auth_info: "{{ caddy_license_account_id }}:{{ caddy_license_api_key }}"
```

Comment:
Maybe `caddy_base_url` and `caddy_auth_info` should be moved into the `defaults.yml` or `vars/main.yml` file?

Reply:
Have done this for `caddy_auth_info` (into `vars/main.yml`).

I didn't do it for `caddy_base_url` because it seemed to end up with a pretty huge jinja2 statement inside the vars (due to it being different depending on whether a license is provided).